spec/forms/support_interface/conditions_form_spec.rb

```diff
@@ -0,0 +1,127 @@
+require 'rails_helper'
+
+RSpec.describe SupportInterface::ConditionsForm do
```

Comment:
This form duplicates some of the logic from `ProviderInterface::OfferWizard`. Can we extract the shared logic into something generic or is that not practical/desirable? (I struggled to see an obvious thing to extract initially but plan to revisit).